src/dotnet/commands/dotnet-run/RunCommand.cs

```diff
@@ -124,11 +124,14 @@ private int RunExecutable()
                 return result;
             }
 
-            // Add Nuget Packages Probing Path
-            var nugetPackagesRoot = _context.PackagesDirectory;
-            var probingPathArg = "--additionalprobingpath";
-            _args.Insert(0, nugetPackagesRoot);
-            _args.Insert(0, probingPathArg);
+            if (!_context.TargetFramework.IsDesktop())
```

Comment:
Can you wrap this in a method? Something like AddAdditionalProbingPathWhenNotDesktop or something like that?